test/unit/points/test-calculate-community-workload-points.js

```diff
@@ -0,0 +1,9 @@
+const expect = require('chai').expect
+const calculateCommunityWorkloadPoints = require('../../../app/points/calculate-community-workload-points')
+
+describe('points/calculate-community-workload-points', function () {
```

Comment:
Is it possible to expand the tests for these calculation classes or is there other stories to come first?